Index.html

```diff
@@ -68,6 +68,243 @@ <h2>I’m Hussen Glad to see you!</h2>
+                      <h2>Back End Dev</h2>
+                     </li>
+                     <li class="frames-list">
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>2015</h2>
+                     </li>
+                 </ul>
+                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 
+                 <ul class="tag-block">
+                  <li class="tag-list">
+                     <h2>html</h2>
+                    </li>
+                     <li class="tag-list">
+                      <h2>css</h2>
+                     </li>
+                     <li class="tag-list">
+                      <h2>javaScript</h2>
+                     </li>
+                 </ul>
+                  <div class="button-acction">
+                      <button class="button-acction-button" type="button"> See Project</button>
+                 </div>
+              </div>
+              </div>
+              </div>
+            </div>
+          </div>
+           <!-- Fourth card  -->
+        <div class="card-work">
+            <div class="card-prtfolio">
+              <img src="img/work-img/Snapshoot Portfolio 4.png" />
+            </div>
+            <div class="work-block">
+              <div class="left-blok">
+                <h2 class="project-title">Tonic</h2>
+                <ul class="frames">
+                  <li class="frames-list">
+                     <h2>CANOPY</h2>
+                    </li>
+                    <li class="frames-list">
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>Back End Dev</h2>
+                     </li>
+                     <li class="frames-list">
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>2015</h2>
+                     </li>
+                 </ul>
+                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 
+                 <ul class="tag-block">
+                  <li class="tag-list">
+                     <h2>html</h2>
+                    </li>
+                     <li class="tag-list">
+                      <h2>css</h2>
+                     </li>
+                     <li class="tag-list">
+                      <h2>javaScript</h2>
+                     </li>
+                 </ul>
+                  <div class="button-acction">
+                      <button class="button-acction-button" type="button"> See Project</button>
+                 </div>
+              </div>
+              </div>
+              </div>
+            </div>
+          </div>
+      </section>
+
+      <section class="my-self-container">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design closely by implementing these changes
- Please kindly reduce the font of `About Us` and increase the line height to make sure it matches the Figma design.
- Kindly implement your button to match the Figma Design


| Your Template  | Design Template |
| ------------- | ------------- |
| <img width="310" alt="image" src="https://user-images.githubusercontent.com/52934128/221383869-ba257107-220f-4122-ae2e-9686fbf5ea68.png"> | <img width="351" alt="image" src="https://user-images.githubusercontent.com/52934128/221383788-59eb24d3-328a-492b-ba7d-4264fff998e7.png">  |